src/overlays/actors/ovl_En_Hint_Skb/z_en_hint_skb.c

```diff
@@ -126,36 +126,36 @@ static DamageTable sDamageTable = {
 };
 
 typedef enum EnHinkSkbAnimation {
-    /*  0 */ ENHINTSKB_ANIM_0,
-    /*  1 */ ENHINTSKB_ANIM_1,
-    /*  2 */ ENHINTSKB_ANIM_2,
-    /*  3 */ ENHINTSKB_ANIM_3,
-    /*  4 */ ENHINTSKB_ANIM_4,
-    /*  5 */ ENHINTSKB_ANIM_5,
-    /*  6 */ ENHINTSKB_ANIM_6,
-    /*  7 */ ENHINTSKB_ANIM_7,
-    /*  8 */ ENHINTSKB_ANIM_8,
-    /*  9 */ ENHINTSKB_ANIM_9,
-    /* 10 */ ENHINTSKB_ANIM_10,
-    /* 11 */ ENHINTSKB_ANIM_11,
-    /* 12 */ ENHINTSKB_ANIM_12,
-    /* 13 */ ENHINTSKB_ANIM_MAX
+    /*  0 */ STALCHILD_ANIM_WALK,
```

Comment:
Actually, because `z_en_skb.c` is the main `STALCHILD` actor, could you rename this enum to `STALCHILD_HINT_ANIM_X` since this is the version of Stalchild's that gives hints

Reply:
Ah, makes sense. Changed all these to STALCHILD_HINT_xyz